### ISSUE TITLE
distributed training？

### ISSUE THREAD

**Issue by weifengchiu** [tensorflow/similarity#252]
Hello, is distributed training supported？
If supported, would you give some examples about that.

**Comment by owenvallis** [comment]
The losses subclass keras.Losses and should support [tf.distribute.Strategy](https://www.tensorflow.org/api_docs/python/tf/distribute/Strategy). I've trained using MultiSim loss and the MirroredStrategy before, but I haven't tested it for all the losses yet.

You can find an example of the MirroredStrategy in the [notebook](https://github.com/tensorflow/similarity/blob/kaggle/examples/supervised/kaggle_keras_tuner.ipynb) on the kaggle branch.  Just a note that the code in that notebook is under development and I haven't touched in a while.
